.github/workflows/java_tests.yml

```diff
@@ -66,7 +66,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [[self-hosted, ubuntu-20.04], macos-latest, [self-hosted, windows-server-2019]]
+        os: [ubuntu-latest, macos-latest, [self-hosted, windows-server-2019]]
```

Comment:
We should probably change this to windows-latest too

Reply:
can do